--- origsrc/pxlib-0.6.3/src/Makefile.am	2007-09-11 09:40:33.000000000 -0500
+++ src/pxlib-0.6.3/src/Makefile.am	2009-04-28 10:51:18.451500000 -0500
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-INCLUDES = -I@srcdir@/../include -I./include -DPACKAGE_LOCALE_DIR=\""$(datadir)/locale"\" @RECODE_INCLUDEDIR@ @GSF_INCLUDEDIR@
+INCLUDES = -I$(top_builddir)/include -I$(top_srcdir)/include -Iinclude -DPACKAGE_LOCALE_DIR=\""$(datadir)/locale"\" @RECODE_INCLUDEDIR@ @GSF_INCLUDEDIR@
 
 noinst_HEADERS = fileformat.h \
                  px_error.h \
