--- origsrc/pxlib-0.6.3/pxlib.pc.in	2003-10-29 12:46:40.000000000 -0600
+++ src/pxlib-0.6.3/pxlib.pc.in	2007-12-22 20:31:20.046875000 -0600
@@ -8,5 +8,5 @@
 Version: @VERSION@
 Requires:
 Libs: -L${libdir} -lpx @RECODE_LIBDIR@ @RECODE_LIBS@ @LIBICONV@
-Cflags: 
+Cflags: -I${includedir}
 
--- origsrc/pxlib-0.6.3/src/Makefile.am	2007-09-11 09:40:33.000000000 -0500
+++ src/pxlib-0.6.3/src/Makefile.am	2007-12-22 20:01:22.375000000 -0600
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-INCLUDES = -I@srcdir@/../include -I./include -DPACKAGE_LOCALE_DIR=\""$(datadir)/locale"\" @RECODE_INCLUDEDIR@ @GSF_INCLUDEDIR@
+INCLUDES = -I../include -I$(srcdir)/../include -I./include -DPACKAGE_LOCALE_DIR=\""$(datadir)/locale"\" @RECODE_INCLUDEDIR@ @GSF_INCLUDEDIR@
 
 noinst_HEADERS = fileformat.h \
                  px_error.h \
@@ -19,7 +19,7 @@
 
 lib_LTLIBRARIES = libpx.la
 
-libpx_la_LDFLAGS = -version-info @PXLIB_VERSION_INFO@
+libpx_la_LDFLAGS = -version-info @PXLIB_VERSION_INFO@ -no-undefined
 
 libpx_la_SOURCES = \
 		paradox.c \
@@ -33,7 +33,7 @@
 		px_crypt.c \
 		gregor.c
 
-libpx_la_LIBADD = @RECODE_LIBS@ @GSF_LIBS@
+libpx_la_LIBADD = $(RECODE_LIBS) $(GSF_LIBS) $(INTLLIBS) $(LTLIBICONV)
 
 BUILD_LIBS = -lm
 
